src/spec/parameter.rs

```diff
@@ -2,6 +2,17 @@ use serde::{Deserialize, Serialize};
 
 use super::{FromRef, Ref, RefError, RefType, Spec};
 use crate::Schema;
+use serde::ser::SerializeStruct;
+
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Default)]
+#[serde(rename_all = "camelCase")]
+pub enum ParamLoc {
+    #[default]
```

Comment:
good change

but.... why is query the default? [the spec](https://spec.openapis.org/oas/v3.1.0.html#parameter-object) doesn't seem to define this